src/app/server.ts

```diff
@@ -26,12 +27,16 @@ import { logger } from './logging';
 
 // this export is the function used by webpackHotServerMiddleware in /scripts/frontend-dev-server
 export default (options: any) => {
-	if ('amp' in options) {
-		return renderAMPArticle;
-	}
-
-	if ('json' in options) {
-		return renderArticleJson;
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+	switch (options.path) {
+		case '/Article':
+			return renderArticle;
+		case '/ArticleJson':
+			return renderArticleJson;
+		case '/AMPArticle':
+			return renderAMPArticle;
+		case '/Interactive':
+			return renderInteractive;
```

Comment:
This will save me about 1 hour whenever I go to add a route 😆

Reply:
Yeah, it's still a bit messy as same info duplicated but better than before at least :)